.github/workflows/build.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     # Consistent with base image in Dockerfile
     # container: stepik/hyperstyle-base:py3.8.11-java11.0.11-node14.17.3
```

Comment:
Please, change to stepik/hyperstyle-base:py3.8.11-java11.0.11-node14.17.3-go1.18.5 🥺